roles/hiring.md

```diff
@@ -0,0 +1,77 @@
+
+That should be your mindset when conducting interviews: you are not searching for an ideal candidate. You are only on the lookout for big "no-no"'s. If someone lacks some skills, they can acquire them. If someone is used to different type of tools or processes, they can learn ours. Look for problem-solving prowess, ability to communicate what went wrong and ownership of mistakes.
+
+### Hiring workflow
+
+Workable guides us through the following stages of hiring a new Nitean. A candidate should expect to wait up to 5 business days between each step of the process.
+
+1. Sourcing
+
+   We post the job ads to various free and paid options, on a per-position basis, but here's a general idea:
+   * Workable posts to 15+ US-based free boards automatically (Glassdoor, Indeed).
+   * We post manually to Slovenian boards: slo-tech.com, internetmojster.com, facebook.com/groups/sluzbezadeveloperje.
+   * We post to twitter.com/teamniteo and facebook.com/niteo.
+   * Sometimes we run an ad campaign on Facebook or buy ads on StackOverflow.
+
+2. Application
+
+   Interested candidates get to our Application Form, hosted by Workable. The form needs to be short, with only general questions. Resist the temptation to ask advanced (technical) questions to asses needed skills for the position. We do these in the next step.
+
+3. Questionnaire
+
+   Members of the hiring team rate (+1/-1) candidates based on application form answers. Whoever gets two or more +1s gets transitioned to the next step: the Questionnaire. Use the `Advanced Questionnaire - <position>` email template to send the questions/tasks to the candidate.
+
+   When the answers arrive we rate the candidate using the scorecard that is prepared in advance and matches the `Advanced Questionnaire - <position>` template. Discussion happens in comments inside Workable, so everything is kept in one place.
+
+   Whoever gets two or more +1s gets transitioned to the next step: Phone Screen.
+
+4. Phone Screen
+
+   A chat over Skype, Hangouts or Zoom with @marbeann. Calls can last anywhere between 10 and 30. The goal is to confirm the candidate is able to communicate via video conferencing. Is their connection OK? Is their English OK? Can they hold a discussion on a subject?
+
+   @marbeann also confirms the following:
+   * Weekly commitment for the trial period (full- or part-time).
+   * When can they start?
+
+5. Interview
+
+   Before the interview we need to select the Nitean who will mentor the candidate should the candidate start the trial period.
+
+   Then the proposed mentor for the candidate and at least one of founders, along with anyone who is interested or will work with the candidate, get on Zoom to have a chat with the candidate.
```

Comment:
`get on Zoom` sounds a bit off, perhaps `create a meeting room on Zoom`

Reply:
we use `get on Zoom` elsewhere